index.html

```diff
@@ -8,6 +8,44 @@
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <div>
+              <a href="#"
+                ><img
+                  class="logo"
+                  src="img/logo-redimensionada.png"
+                  alt="Logo coding cerrado"
+              /></a>
+            </div>
+          </li>
+          <li>
+            <div>
+              <a class="nav-title" href="#"><strong>Coding Cerrado</strong></a>
+              Editor
+            </div>
+          </li>
+        </ul>
+
+        <ul class="right-content">
+          <li>
+            <div><a class="nav-hover" href="#">Resume</a></div>
+          </li>
+          <li>
+            <div class="border-left">
```

Comment:
```suggestion
            <div class="link-with-border">
```
Evite usar uma palavra reservada como nome de classe (border-left eh o nome do proprio comando que vai adicionar a borda). Lembre de atualizar o nome da classe no css tb.

Reply:
Sim, @jhrodriguesit !
Valeu por mais essa dica!